wiki/en/en-Tips-Tricks-More.md

```diff
@@ -98,18 +98,10 @@ jack_connect Jamulus:'output right' system:playback_2
 
 
 
-## Using the --ctrlmidich MIDI controller channel option
+## Using --ctrlmidich for MIDI controllers
```

Comment:
Jekyll automatically translates `--` to `–` It should be `--` --> the command will not work on the cli if copied. Cold we maybe change the heading to something like Using ctrlmidich for MIDI controllers
and start the text with something like 

The volume faders in the client's mixer window can be controlled using a MIDI controller and the parameter `--ctrlmidich`

Reply:
Done.

I was aware of the discussion about what to implement for MIDI controllers but didn't see it was included in the release until yesterday (it was merged on Sunday). I also had to figure out how to actually use the new parameter because it isn't explicitly documented anywhere - hence why I saw the need to document it myself.